fsspec/parquet.py

```diff
@@ -91,20 +120,35 @@ def open_parquet_file(
         footer_sample_size=footer_sample_size,
     )
 
+    # If data is empty, just use default
+    # `open` command with `path` input
+    if len(data) != 1:
+        return fs.open(path, mode=mode)
```

Comment:
I'm not sure you want this shortcut. You still might have multiple requests sequentially if the one block is big compared to the readahead size (no opportunity to pass other arguments here)

Reply:
Your are right - Removed.